telegram/_chatpermissions.py

```diff
@@ -121,3 +121,23 @@ def __init__(
             self.can_invite_users,
             self.can_pin_messages,
         )
+
+    @classmethod
+    def all_permissions(cls) -> 'ChatPermissions':
+        """
+        This method returns the ChatPermissions object with all attributes set to :obj:`True`.
+        This is e.g. useful when unrestricting a ChatMember with
```

Comment:
either "chat member" or link to the class

Reply:
I like the second one.